sdk/python/feast/feature_table.py

```diff
@@ -43,18 +44,28 @@ class FeatureTable:
     def __init__(
         self,
         name: str,
-        entities: Union[str, List[str]],
-        features: Union[FeatureV2, List[FeatureV2]],
-        batch_source: Optional[DataSource] = None,
-        stream_source: Optional[DataSource] = None,
+        entities: List[str],
+        features: List[Feature],
+        batch_source: Union[BigQuerySource, FileSource] = None,
+        stream_source: Optional[Union[KafkaSource, KinesisSource]] = None,
         max_age: Optional[Duration] = None,
         labels: Optional[MutableMapping[str, str]] = None,
     ):
         self._name = name
         self._entities = entities
-        self._features = features
-        self._batch_source = batch_source
-        self._stream_source = stream_source
+        self._features = [
```

Comment:
Can you explain what is happening here?

Reply:
Yup, so when generating FeatureTableSpec in `to_proto` method, it requires `self.features` in proto form, this is because we're expecting `FeatureSpecV2` object as defined in FeatureTable.proto file.

Reply:
I've cleaned this up, to shift conversion inside `to_proto` method, this makes typings and initialization of class object code alot cleaner.